lib/backend/k8s/resources/watcher_test.go

```diff
@@ -0,0 +1,229 @@
+// Copyright (c) 2020 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resources
+
+import (
+	apiv3 "github.com/projectcalico/libcalico-go/lib/apis/v3"
+	"github.com/projectcalico/libcalico-go/lib/backend/api"
+	"github.com/projectcalico/libcalico-go/lib/backend/model"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	log "github.com/sirupsen/logrus"
+	kwatch "k8s.io/apimachinery/pkg/watch"
+)
+
+var _ = Describe("Resources watcher ", func() {
+	Describe("convertEvent tests", func() {
+
+		kwc := k8sWatcherConverter{
```

Comment:
I think you should either just create this in each test, or create this in a BeforeEach function, otherwise changes to this watcherConverter in one tests will propagate to the next tests that are run.